src/script.js

```diff
@@ -0,0 +1,58 @@
+const days = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
+
+function createCalendar(id, year, month) {
+    const containerForTable = document.querySelector(`#${id}`);
+    const table = document.createElement('table');
+    const thead = createTableHead(days);
+    table.appendChild(thead);
+    const dataMonth = getDataMonth(year, month);
+    const tbody = createTableBody(dataMonth);
```

Comment:
will be better to write all const then empty line then appends, or separate logic parts(create head, body, append to container) by empty lines